### ISSUE TITLE
RazorGenerator.Mvc.dll is not extracted to package folder

### ISSUE THREAD

**Issue by frankzdd** [RazorGenerator/RazorGenerator#69]
Try adding the latest RG version 2.3.12 to a MVC project with .Net framework 4.6.1, the RazorGenerator.Mvc.dll file is not extracted to package folder and the reference to it is not set either.

Notice the lib folder is missing in the package folder:

![rg](https://cloud.githubusercontent.com/assets/16503020/12066438/ec48f8ee-afb5-11e5-87ac-0247b0f49f82.png)

Then I tried installing version 2.3.11, it extracted correctly with the lib folder. I even changed the target .Net framework version to a different one, it is all the same. Btw, this is on a fresh installed Windows 10 Enterprise + Visual Studio 2015 Enterprise machine.

**Comment by davidebbo** [comment]
I just tried and could not repro. My guess is that you had something else going on that had nothing to do with 2.3.11 vs 2.3.12.

Please use source control to get back to a clean state before to was added, and try doing it again with each version.

**Comment by frankzdd** [comment]
Thanks @davidebbo, I finally figured it out.

For whatever reason, the first download file of package 2.3.12 was corrupted or who knows what happened. It was cached in "C:\Users\[user.domain]\.nuget\packages\RazorGenerator.Mvc\2.3.12" folder. The lib\net40 folder is there but the dll file is missing. I guess when the package was extracted, the dll file was corrupted so it did not get extracted. Or whatever anti-virus on my network filtered that binary file out.

I removed that 2.3.12 folder, uninstalled RG from my project, restarted VS, and installed it again. Now it works perfectly.

I think if possible the init.ps1 should do some verification once it loads the package into the project, and prompt if anything is missing or not setup correctly. Just my thought.
